From cfff8198356afd9b9a641dd1e54d4b0b2a35e6d3 Mon Sep 17 00:00:00 2001
From: Anish Trivedi <anish@freescale.com>
Date: Wed, 15 Sep 2010 10:26:36 -0500
Subject: [PATCH 178/442] ENGR00131570 SRTC Driver enable for kernel 2.6.35

Skip RTC time set notification test if kernel version is
2.6.35.3. The WAIT_TIME_SET ioctl in the SRTC driver is not
functioning as it did on kernel 2.6.31.

Signed-off-by: Anish Trivedi <anish@freescale.com>
---
 test/mxc_rtc/autorun-rtc.sh | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/test/mxc_rtc/autorun-rtc.sh b/test/mxc_rtc/autorun-rtc.sh
index ce327fe..2bde916 100755
--- a/test/mxc_rtc/autorun-rtc.sh
+++ b/test/mxc_rtc/autorun-rtc.sh
@@ -39,8 +39,13 @@ if [ "$RTC_IRQS" != "$RTC_IRQS_EXPECTED" ]; then
 	STATUS=1
 else
 	echo "checking rtc interrupts PASS"
+
+	# Time set notification mechanism has not yet been ported to 2.6.35
+	KERNEL_VERSION=$( cat /proc/version|cut -d' ' -f3|cut -d'-' -f1)
+	if [ "$KERNEL_VERSION" != 2.6.35.3 ]; then
        #RTC wait for time set notification test
         run_testcase "./rtc_timesetnotification_test.out"
+      fi
 fi
 
 print_status
-- 
1.8.0

